src/patternfly/components/Drawer/drawer.scss

```
@@ -479,13 +479,9 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
     }
 
     > .pf-c-drawer__main > .pf-c-drawer__panel.pf-m-resizable {
-      padding-top: var(--pf-c-drawer--m-panel-bottom__panel--m-resizable--PaddingTop);
-      padding-left: 0;
+      flex-direction: var(--pf-c-drawer--m-panel-bottom__panel--m-resizable--FlexDirection);
```

We try to reassign a var if we can instead of overriding the property. However, CSS vars cascade. So you can introduce them at the highest level selector where you need the change. It looks like you might be able to make this variable re-assignment a few lines up, underneath `.pf-c-drawer.pf-m-panel-bottom`

```suggestion
      --pf-c-drawer__panel--m-resizable--FlexDirection: var(--pf-c-drawer--m-panel-bottom__panel--m-resizable--FlexDirection);
```